lib/ome/common/module.cpp

```diff
@@ -76,9 +76,9 @@ namespace
     // Global paths (not specific to any component)
     static RegisterModule bin
       ("bin",
-       "BIOFORMATS_BINDIR",
+       "OME_FILES_BINDIR",
```

Comment:
The reason why it doesn't use it above is because we haven't refactored it to do so yet.  As I mentioned above, it's planned.  The existing state is due to it retaining the behaviour of the set-up prior to the split in some aspects; before the split there was only one set of paths which were used by all components.  The stuff merged over the last two days moves component-specific paths into their respective components.  The next bit will fix this up, as I already said.

The reason why it's here right now is that this is the root component.  There is only one `bin` directory, and that is the ome-files one.  Every component uses this (or will fall back to it, post-refactoring).